adapters/Vungle/Public/Headers/VungleRouterConsent.h

```diff
@@ -13,9 +13,16 @@
 // limitations under the License.
 
 #import <Foundation/Foundation.h>
-typedef NS_ENUM(NSInteger, VungleConsentStatus);
 
 @interface VungleRouterConsent : NSObject
-+ (BOOL)updateConsentStatus:(VungleConsentStatus)consentStatus;
-+ (VungleConsentStatus)getConsentStatus;
+
+/// Updates the GDPR status in the Vungle SDK
++ (void)updateGDPRStatus:(BOOL)optIn;
+
+/// Updates the CCPA status in the Vungle SDK
++ (void)updateCCPAStatus:(BOOL)optIn;
+
+/// Determines whether the IDFV can be collected or not
++ (void)updateIDFVStatus:(BOOL)optIn;
+
```

Comment:
When should these consent values be passed to the Vungle SDK? Is it before Vungle SDK initializes or at ad request time?

Reply:
It can be anytime. But for it to take effect, it needs to happen before the ad request is made.